docs/sources/determine-use-case/index.md

```diff
@@ -0,0 +1,36 @@
+---
+description: Determine your use case to begin your investigation
+keywords:
+  - Explore Profiles
+  - Use case
+title: Determine your use case
+menuTitle: Determine your use case
+weight: 400
+---
+
+# Determine your use case
+
+When you start investigating an issue, you may either know what's wrong (for example, you know the affected service or that there’s too much CPU usage), or you may want to explore data.
+This can lead you to two different starting points:
+
+* Use case 1 - You know which service has issues and you need to investigate why.
+* Use case 2 - You know there's a problem, but you need to locate where it is and the cause
+
+Your use case determines what’s most important. If a service is misbehaving, then you might want to see the profile types so you can see the CPU and memory profiles alongside each other.
+
+For either use case, the first step is to identify areas of interest by reviewing profiles or a single service.
+Selecting different profile types lets you focus on memory allocation, CPU processes, allocation sizes, blocks, and more.
```

Comment:
```suggestion
Selecting different profile types lets you focus on memory allocation, CPU processes, allocation sizes, and blocks.
```

Reply:
This is another case where there are more profile types available. The trick here is that the profile type you can access depends upon how you've instrumented profiling in your app. here's the list of profiling types and the supported instrumentations: https://grafana.com/docs/pyroscope/latest/view-and-analyze-profile-data/profiling-types/

There isn't a subset that is available in all profile types, only CPU. 

What about this?  
```suggestion
Selecting different profile types lets you focus on memory allocation, CPU processes, allocation sizes, blocks, or blocks. 
The profile types available depend on your profiling data.
For more information, refer to [Understand profile types](https://grafana.com/docs/pyroscope/latest/view-and-analyze-profile-data/profiling-types/).
```